tests/test_blocks.py

```diff
@@ -10,7 +10,7 @@
     CodeBlock, BulletList, OrderedList, Decimal, Period, Meta, Pandoc, Link,
     HorizontalRule, BlockQuote, Image, MetaString, Table, TableHead, TableBody,
     TableFoot, RowHeadColumns, Row, Cell, RowSpan, ColSpan, ColWidthDefault, AlignDefault,
-    Caption, Math, DisplayMath
+    Caption, Math, DisplayMath, 
```

Comment:
I think flake8 will catch this, but you have some trailing whitespace here.

Reply:
Got it.